### PR TITLE
Somebody once told me the world is gonna roll me

### PR DESCRIPTION
**Pull request by puppy0cam**
She was looking kind of dumb trying to declare their own shit, so the documentation shall be changed to fix some levels of stupidity

### CODE REVIEW

**Review by sad-unicorn — CHANGES_REQUESTED** [review]
The thing is, people are not aware that they are declaring. The question is more likely to be 'Why I am getting ACCESS_REFUSED error' 

And there is a third question 'Why I can't login' with answer 'you are trying to access the wrong API instance'

File: index.md [hunk 1]
```diff
@@ -5,10 +5,16 @@ navigation: 1
...
 
 ## Abstract
-Interaction is done via AMQP (by rabbitMQ). Application should publish a message to the appropriate exchange with  an appropriate routing key, and then, listen for and react to incoming messages in its queue. E.g. for the application named `dices` there would a be direct exchange named `dices_ex`, routing key `dices_o` and an inbound queue `dices_i` (for the testing purpose there will be `dices_to` and `dices_ti` queues respectively)
+Interaction is done via AMQP (by rabbitMQ). Application should publish a message to the appropriate exchange with  an appropriate routing key, and then, listen for and react to incoming messages in its queue. E.g. for the application named `dices` that is running on the EU server there would a be direct exchange named `dices_ex`, routing key `dices_o` and an inbound queue `dices_i` (for the testing purpose there will be `dices_to` and `dices_ti` queues respectively)
```
Comment (sad-unicorn) [inline index.md]: it's not EU, it's international

File: index.md [hunk 2]
```diff
@@ -18,3 +24,97 @@ Apart from that, there is a set of public fan-out read-only exchanges, where you
...
 
 Please note, that at this moment, API is highly unstable.
+
+### Connection
+
+There are two versions of the game with an API, and there are two versionf of the API. Refer to this table for what API to connect to. Be aware that an app will not automatically have the same access to both APIs, and you must request access to each API seperately.
+
+| Server | URL |
+| --- | --- |
+| EU | amqps://api.chatwars.me:5673 |
```
Comment (sad-unicorn) [inline index.md]: no

File: index.md [hunk 3]
```diff
@@ -18,3 +24,97 @@ Apart from that, there is a set of public fan-out read-only exchanges, where you
...
+A: You are not meant to be declaring anything within the API, You only have to consume and publish.
+
+---
+Q: **Why can't I log in?**
+
+A: You may not be connecting to the correct instance of the API! If you wish to connect to the CW3 API, This table is a good reference for what API corresponds to what URL
+
+| Server | URL |
+| --- | --- |
+| EU | amqps://api.chatwars.me:5673 |
```
Comment (sad-unicorn) [inline index.md]: wronk

File: index.md [hunk 4]
```diff
@@ -18,3 +24,97 @@ Apart from that, there is a set of public fan-out read-only exchanges, where you
 For now all queues are limited in size by 1Mb, with discard-oldest policy. 
 
 Please note, that at this moment, API is highly unstable.
+
+### Connection
+
+There are two versions of the game with an API, and there are two versionf of the API. Refer to this table for what API to connect to. Be aware that an app will not automatically have the same access to both APIs, and you must request access to each API seperately.
```
Comment (sad-unicorn) [inline index.md]: typo 'versionf'

File: index.md [hunk 5]
```diff
@@ -18,3 +24,97 @@ Apart from that, there is a set of public fan-out read-only exchanges, where you
...
+
+| Server | URL |
+| --- | --- |
+| EU | amqps://api.chatwars.me:5673 |
+| CW3 | amqps://api.chtwrs.com:5673 |
+
+## FAQ
+Q: **Why can I not declare my queues?**
+
+A: Because they are pre-declared for you. You only have to consume and publish
```
Comment (sad-unicorn) [inline index.md]: the real answer is 'security reasons', current answer is wrong

File: index.md [hunk 6]
```diff
@@ -18,3 +24,97 @@ Apart from that, there is a set of public fan-out read-only exchanges, where you
...
+| EU | amqps://api.chatwars.me:5673 |
+| CW3 | amqps://api.chtwrs.com:5673 |
+
+---
+Q: **How do I obtain access to the API?**
+
+A:
+
+To access the Chat Wars API, you must first obtain a login from the feedback bot.
+* To obtain access to [@chtwrsbot (EU)](http://t.me/chtwrsbot)'s API, You must request a login from [@ChatWarsFeedbackBot](http://t.me/ChatWarsFeedbackBot)
```
Comment (sad-unicorn) [inline index.md]: int

**Review by sad-unicorn — CHANGES_REQUESTED** [review]

**Review by sad-unicorn — APPROVED** [review]
its aight

**Comment by louietyj** [comment]
Lel 100 x "Update changes.md"...

**Pull request merged**